src/taskWifi.cpp

```diff
@@ -16,7 +23,16 @@ void TaskWifi(void* pvParameters) {
   WiFi.mode(WIFI_STA);
   // WiFi.begin(ssid, password);
   WiFi.setTxPower(WIFI_POWER_8_5dBm);
-
+ // WiFi.disconnect(true);
+        //
+// if identity not defined use ursername as identity
```

Comment:
But does it often happens that we need an identify different from a username ? Just curious.